pkg/test/framework/components/istio/operator.go

```diff
@@ -128,25 +138,94 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		return i, nil
 	}
 
-	istioCtl, err := istioctl.New(ctx, istioctl.Config{})
+	// Create the system namespace in all the clusters.
+	if _, err := namespace.Claim(ctx, cfg.SystemNamespace); err != nil {
+		return nil, err
+	}
+
+	if i.environment.IsMulticluster() {
+		// Copy the CA secrets to all clusters so that they all share the same root of trust.
+		for _, cluster := range env.KubeClusters {
+			if err := cluster.CreateSecret(cfg.SystemNamespace, caSecrets); err != nil {
+				return nil, fmt.Errorf("failed to create CA secrets on cluster %d: %v", cluster.Index(), err)
+			}
+		}
+
+		// Configure direct access for each control plane to each APIServer. This allows each control plane to
+		// automatically discover endpoints in remote clusters.
+		for clusterIndex, cluster := range env.KubeClusters {
+			// Create a secret.
+			secret, err := createRemoteSecret(ctx, cluster)
+			if err != nil {
+				return nil, err
+			}
+
+			// Copy this secret to all control plane clusters.
+			for remoteIndex, remote := range env.ControlPlaneClusters() {
+				if clusterIndex != remoteIndex {
+					if _, err := remote.ApplyContents(cfg.SystemNamespace, secret); err != nil {
+						return nil, fmt.Errorf("")
+					}
+				}
+			}
+		}
```

Comment:
These lines need to be moved after the control plane is deployed. I forgot that we need the `istio-reader-service-account` to be created before we can create the secrets.